src/communication/mpi.hpp

```diff
@@ -65,10 +65,6 @@ namespace mpi {
     // T must be trivially copyable
     template<typename T>
     std::vector<T> gather(T value, int root) {
-        static_assert(
-            true,//std::is_trivially_copyable<T>::value,
-            "gather can only be performed on trivally copyable types");
-
```

Comment:
For the public record: it was apparently me who originally commented this out. I have no recollection as to why, though I'm hoping it was just some debugging test that slipped inadvertently into a commit.